backend/src/zac/core/api/views.py

```diff
@@ -376,3 +386,85 @@ def group_key(zo):
 
         serializer = self.serializer_class(instance=groups, many=True)
         return Response(serializer.data)
+
+
+class ZaakTypenView(ListAPIView):
+    authentication_classes = (authentication.SessionAuthentication,)
+    permission_classes = (permissions.IsAuthenticated,)
+    serializer_class = ZaakTypeAggregateSerializer
+    pagination_class = BffPagination
+    filter_backends = (ApiFilterBackend,)
+    filterset_class = ZaaktypenFilterSet
+    action = "list"
+
+    def get_queryset(self) -> List[dict]:
+        zaaktypen = self.get_zaaktypen()
+        return zaaktypen
+
+    def get_zaaktypen(self) -> List[dict]:
+        zaaktypen = get_zaaktypen(UserPermissions(self.request.user))
+
+        # aggregate
+        zaaktypen_data = [
+            {
+                "catalogus": zaaktype.catalogus,
+                "identificatie": zaaktype.identificatie,
+                "omschrijving": zaaktype.omschrijving,
+            }
+            for zaaktype in zaaktypen
+        ]
+        zaaktypen_aggregated = [
+            dict(z) for z in set(tuple(zaaktype.items()) for zaaktype in zaaktypen_data)
+        ]
```

Comment:
You could also use frozenset for this:

```suggestion
        zaaktypen_aggregated = {
            frozenset(zaaktype.items()) : zaaktype for zaaktype in zaaktypen_data
        }.values()
```